staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -222,6 +222,10 @@ const (
 	// document.
 	StorageVersionHash featuregate.Feature = "StorageVersionHash"
 
+	// owner: @serathius
+	// Allow apiserver to encode List objects item by item instead of all at once.
+	StreamingJSONListEncoding featuregate.Feature = "StreamingJSONListEncoding"
```

Comment:
nit: the API is **list**; the thing we're encoding is a _collection_
```suggestion
	StreamingCollectionEncodingToJSON featuregate.Feature = "StreamingCollectionEncodingToJSON"
```
Also, I expect we might have a YAML and / or CBOR feature gate one day alongside this, so I changed the structure of the feature gate name to make that easier to work with.

Reply:
See https://kubernetes.io/docs/reference/using-api/api-concepts/#collections for the definition of a collection. I think it's not a great choice of name, but it's what we use.

Reply:
Done